commands/verify/verify.go

```diff
@@ -101,23 +103,37 @@ func DoVerifyAccordingConfig() error {
 		return err
 	}
 
+	failFast := e2eConfig.Verify.FailFast
+
+	var errMsgList []string
```

Comment:
@kezhenxu94 How about using `multierror`? 👇
https://github.com/hashicorp/go-multierror

Reply:
Good idea.

Reply:
OK! I will learn the "multierror" right now!